packages/driver/src/dom/visibility.js

```diff
@@ -112,6 +116,42 @@ const elHasVisibilityHidden = ($el) => {
   return $el.css('visibility') === 'hidden'
 }
 
+// This is a simplified version of backface culling ().
+// https://en.wikipedia.org/wiki/Back-face_culling
+//
+// We defined view normal vector, (0, 0, -1), - eye to screen.
+// and default normal vector, (0, 0, 1)
+// When dot product of them are >= 0, item is visible.
+const elIsBackface = ($el) => {
+  const el = $el[0]
+  const style = getComputedStyle(el)
+  const backface = style.getPropertyValue('backface-visibility')
+  const backfaceInvisible = backface === 'hidden'
+
+  if (backfaceInvisible) {
+    const transform = style.getPropertyValue('transform')
+
+    if (transform.startsWith('matrix3d')) {
+      const m3d = transform.substring(8).match(/-?[0-9]+(?:\.[0-9]+)?/g)
```

Comment:
Pull this regex out to the root level into a `const` so it doesn't get re-instantiated every time this function is run and so it's given a name.